src/smt/smt_engine.cpp

```diff
@@ -2535,8 +2580,11 @@ void SmtEngine::checkUnsatCore() {
   Notice() << "SmtEngine::checkUnsatCore(): generating unsat core" << endl;
   UnsatCore core = getUnsatCore();
 
-  SmtEngine coreChecker(d_exprManager);
+  SmtEngine coreChecker(d_exprManager, &d_options);
+  coreChecker.setIsInternalSubsolver();
   coreChecker.setLogic(getLogicInfo());
+  coreChecker.setOption("check-unsat-cores", false);
```

Comment:
Has this change (of disabling unsat cores and proof) anything to do with this PR?

Reply:
This was done before, but in a more roundabout way (see diff).